rancher/v1.2/en/cattle/adding-load-balancers/index.md

```diff
@@ -104,47 +86,25 @@ When there are multiple hostname routing rules, the order of priority is as foll
 5. URL
 6. Default (no hostname, no URL)
 
-##### Target Port
+##### Target Service and Port
 
-For each service, you can override the target port defined in the listening port section. This would allow you to direct traffic to services on different ports from the same source port.
+For each rule, you select the specific **target** service to direct traffic to. Along with the service, you select the **port** for the service. This private port on the service is typically the exposed port on the image.
 
-**Example**
+#### Selector rule
 
-Source Port| Default Target Port
----|---
-80 | 80
+Selector rules allow you to add services into Rancher after the load balancer has been created. Services matching the selector will become a target in the load balancer.
```

Comment:
You can add services to the LB even the regular way, after the LB had been created. What selector rule is really useful for - it allows you to NOT define the service links/hostname routing rules on the LB, but rather define them on the target service level.